app/views/layouts/application.html.erb

```diff
@@ -24,7 +24,7 @@
           <!-- collapse navbar -->
             <a><%= link_to "Vagabond", root_path, :class =>"navbar-brand",:id=>"header" %></a>
           </div>
-          <div class="collapse navbar-collapse">
+          <div class="collapse navbar-collapse" id="myNavbar">
```

Comment:
Prefer lower-kabob-case to camelCase.
https://google.github.io/styleguide/htmlcssguide.html#Capitalization